sdk/azcore/log.go

```diff
@@ -6,62 +6,41 @@
 package azcore
 
 import (
-	"github.com/Azure/azure-sdk-for-go/sdk/internal/logger"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/log"
 )
 
-// LogClassification is used to group entries.  Each group can be toggled on or off
-type LogClassification logger.LogClassification
+// LogClassification is used to group entries.  Each group can be toggled on or off.
+type LogClassification = log.Classification
 
 const (
 	// LogRequest entries contain information about HTTP requests.
 	// This includes information like the URL, query parameters, and headers.
-	LogRequest LogClassification = "Request"
+	LogRequest = log.Request
```

Comment:
You'll get stuttering here: log.LogRequest. How about RequestClassification, etc?

Reply:
This will be fixed in a subsequent PR to move logging to its own package (will post shortly).